syntax = "proto3";

import "google/protobuf/timestamp.proto";

package bakdata.corporate.v1;

enum Status {
  STATUS_UNSPECIFIED = 0;
  STATUS_INACTIVE = 1;
  STATUS_ACTIVE = 2;
}

enum Role {
  ROLE_CEO = 0;
}

message Contact {
  string address = 1;
  string phone = 2;
  string email = 3;
  string website = 4;
}

message Range {
  uint32 start = 1;
  uint32 end = 2;
}

message GrantDonation {
  string name = 1;
  string location = 2;
  Range money = 3;
  string project = 4;
}